src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/moderation/AttributeModifierWrapper.java

```diff
@@ -0,0 +1,18 @@
+package pl.tomaszdziurko.jvm_bloggers.view.admin.moderation;
+
+import org.apache.wicket.AttributeModifier;
+import org.apache.wicket.behavior.AttributeAppender;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+
+/**
+ * @autor mszarlinski
+ */
+@Component
+public class AttributeModifierWrapper {
```

Comment:
There is more "Wicket" way to test this without any additional classes.

`AttributeModifier.append()` static method is actually creating new `AttributeModifier` with given key and value to the list of Behaviours of Item.

So you can verify what was added by checking following stuff:

    item.getBehaviors(AttributeAppender.class).get(0).getAttribute();
    item.getBehaviors(AttributeAppender.class).get(0).replaceModel.getObject(); 

Unfortunately `replaceModel` is protected but with groovy you will be able to get it without much hassle and then you can verify if found object has proper values set.